packages/core/src/stream-loading/__tests__/tip-fetcher.test.ts

```diff
@@ -44,7 +46,11 @@ describe('TipFetcher test', () => {
       if (message.typ == MsgType.QUERY && message.stream.equals(streamID)) {
         const tipMap = new Map().set(streamID.toString(), tip)
         const response: PubsubMessage = { typ: MsgType.RESPONSE, id: message.id, tips: tipMap }
-        ipfs2.pubsub.publish(TOPIC, serialize(response))
+        ipfs2.pubsub.publish(TOPIC, serialize(response)).catch((error) => {
+          logger.err(
```

Comment:
hmm... I don't think it really makes sense to use the logger like this in test code.  If any of this fails we want the test to fail, not for the error to be swallowed with a message to the logs.  Can we remove the catch statement and leave the test code as it was?

Reply:
catching the error, re-throwing it and adding a comment to explain